EventsExpress/ClientApp/src/components/helpers/helpers.js

```diff
@@ -18,253 +18,257 @@ import Input from '@material-ui/core/Input';
 import MenuItem from '@material-ui/core/MenuItem';
 
 export const radioButton = ({ input, ...rest }) => (
-  <FormControl>
-    <RadioGroup {...input} {...rest}>
-      <FormControlLabel value="blocked" control={<Radio />} label="Blocked" />
-      <FormControlLabel value="unblocked" control={<Radio />} label="Unblocked" />
-      <FormControlLabel value="all" control={<Radio />} label="All" />
-    </RadioGroup>
-  </FormControl>
+    <FormControl>
+        <RadioGroup {...input} {...rest}>
+            <FormControlLabel value="blocked" control={<Radio />} label="Blocked" />
+            <FormControlLabel value="unblocked" control={<Radio />} label="Unblocked" />
+            <FormControlLabel value="all" control={<Radio />} label="All" />
+        </RadioGroup>
+    </FormControl>
 )
 
 
 export const validate = values => {
-  const errors = {}
-  const requiredFields = [
-    'email',
-    'password',
-    'RepeatPassword',
-    'title',
-    'description',
-    'categories',
-    'countryId',
-    'cityId',
-    'RepeatPassword',
-    'oldPassword',
-    'newPassword',
-    'repeatPassword',
-    'Birthday',
-    'UserName'
-  ]
-  requiredFields.forEach(field => {
-    if (!values[field]) {
-      errors[field] = 'Required'
+    const errors = {}
+    const numberFields = ['maxParticipants']
+    const requiredFields = [
+        'email',
+        'password',
+        'RepeatPassword',
+        'title',
+        'description',
+        'categories',
+        'countryId',
+        'cityId',
+        'RepeatPassword',
+        'oldPassword',
+        'newPassword',
+        'repeatPassword',
+        'Birthday',
+        'UserName'
+    ]
+
+    requiredFields.forEach(field => {
+        if (!values[field]) {
+            errors[field] = 'Required'
+        }
+    })
+
+    numberFields.forEach(field => {
+        if (values[field] && !/^[1-9]+$/i.test(values[field])) {
```

Comment:
This regex does not allow inputs of 10, 20, 30, 100, etc. Please consider not using regex, but actually using numeric html input.

Reply:
This part of code was deleted.